lib/corefoundation.rb

```diff
@@ -1,6 +1,9 @@
+# frozen_string_literal: true
+
 require 'ffi'
-require 'iconv' if RUBY_VERSION < "1.9"
+require 'iconv' if RUBY_VERSION < '1.9'
```

Comment:
we're not supporting < 1.9 so this can go

Reply:
yes! Will get rid of a couple of hacks from the old code 😄

Reply:
resolved in https://github.com/chef/corefoundation/pull/10/commits/8071e13d1f6e0cbd67b45b2b2f0a70db00a5e6ee